core-web/apps/dotcms-ui/src/app/view/directives/dot-field-required/dot-field-required.directive.ts

```diff
@@ -0,0 +1,20 @@
+import { Directive, ElementRef, Input, OnInit, Renderer2 } from '@angular/core';
+import { FormGroupDirective } from '@angular/forms';
+
+@Directive({
+    selector: '[dotFieldRequired]'
+})
+export class DotFieldRequiredDirective implements OnInit {
+    @Input() for: string;
+
+    constructor(
+        private el: ElementRef,
+        private fg: FormGroupDirective,
+        private renderer: Renderer2
+    ) {}
+    ngOnInit() {
+        if (this.fg?.form.controls[this.for]?.errors?.required) {
```

Comment:
Is there a reason we are checking the `errors` property? I mean, we need to know if the control is `required` so I think something like this might work:

```suggestion
        if (this.fg.form.controls[this.for].hasValidator(Validators.required)) {
```

Reply:
Note